HPXMLtoOpenStudio/resources/EPvalidator.rb

```diff
@@ -525,6 +545,7 @@ def self.run_validator(hpxml_doc)
 
       ## [HVACDistType=Air]
       '/HPXML/Building/BuildingDetails/Systems/HVAC/HVACDistribution/DistributionSystemType/AirDistribution' => {
+        # TODO: Resolve ConditionedFloorAreaServed and DuctLocation outside of unit scope for hvac distribution connected to a preconditioning hvac equipment.
```

Comment:
Some inputs in AirDistribution seem to be hard to frame to preconditioning equipment, eg. ConditionedFloorAreaServed, DuctLocation (can be somewhere outside unit). I think it requires some pre-processing or defaulting in preconditioning context. 
The ConditionedFloorAreaServed is a required element, and I only found it used for defaulting duct surface areas and duct locations, maybe we can: 1. proportion the conditionedfloorarea by the preconditioning cooling/heating fractions(whichever is larger) or 2. take the remaining conditioned floor area if other hvac distributions are all air distributions? 
@shorowit Any thoughts on how to tweak duct locations?